storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -5019,17 +5019,14 @@ static bool xtrabackup_init_temp_log(void) {
 
   uint64_t file_size;
 
-  lsn_t max_no;
   lsn_t max_lsn = 0;
-  lsn_t checkpoint_no;
```

Comment:
I believe we still need this to determine the right checkpoint_header to use. Or not?
max checkpoint_lsn is sufficient I guess. (from 8.0.30 checkoint_no is definitely not used)

@altmannmarcelo may be you can double-check if checkpoint_no is required for pre 8030 or not?

Reply:
I figured it out. We do not use checkpoint_no to determine which checkpoint_lsn to consider.
We just take the highest.